### PR TITLE
added missing spanish translation files

### PR DESCRIPTION
**Pull request by a0viedo**
@thehazzard @eafelix @julianduque any feedback would be appreciated

### CODE REVIEW

File: i18n/troubleshooting_es.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+# Ups, algo no está funcionando.
+# ¡No desesperes!
```
Comment (esalinas00) [inline i18n/troubleshooting_es.md]: I liked # Pero !Que no cunda el pánico!
it was kinda like an easter egg for el Chapulín Colorado watchers.

File: i18n/troubleshooting_es.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+%attempt%
+
+`Diferencia
+===================`
+
+%diff%
+
+## Ayuda adicional:
+ * Has escrito el nombre del archivo correctamente? Puedes comprobarlo ejecutando ls `%filename%`, si tu ves ls: cannot access `%filename%`: No such file or directory entonces deberías crear un nuevo archivo o cambiar de directorio al que contiene tu archivo
```
Comment (esalinas00) [inline i18n/troubleshooting_es.md]: maybe you forgot the "¿"

Comment (gaboesquivel) [inline i18n/troubleshooting_es.md]: @esalinas00 we don't use starting question mark in spanish... just the one at the end :)

Comment (a0viedo) [inline i18n/troubleshooting_es.md]: @esalinas00 is gramatically correct. that being said, I'm in favor of following a convention: every question with both question marks or every question with only one at the end.

**Comment by a0viedo** [comment]
following this on https://github.com/sethvincent/javascripting/pull/98 based in @esalinas00 suggestions

**Pull request closed without merging**